src/main/kotlin/turku/forge/babyproject/api/RootApiController.kt

```diff
@@ -1,19 +1,75 @@
 package turku.forge.babyproject.api
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include
+import org.springframework.hateoas.LinkRelation
+import org.springframework.hateoas.server.core.Relation
+import org.springframework.hateoas.server.mvc.WebMvcLinkBuilder
+import org.springframework.hateoas.server.mvc.linkTo
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import turku.forge.babyproject.config.STOMP_ENDPOINT
+import turku.forge.babyproject.resources.Resource
 
-const val API_PATH = "/"
+// The request path for the root of the API
+const val API_PATH = ""
+
+// Relation constants
+const val WS_RELATION = "ws:connect"
```

Comment:
Using a verb as a link relation is usually a nono, would `endpoint` or `connection-url` be more describing.

Reply:
Yea you are correct about that. I rember reading about the verb thing at some point. I think somthing like `sockjs-endpoint` would work best in this situation, because its a link to the sockjs endpoint 😅